gui/wxpython/startup/utils.py

```diff
@@ -21,38 +21,47 @@
 
 
 def get_possible_database_path():
-    """Finds a path to what is possibly a GRASS Database. 
-
-    Looks for directory named grassdata in the usual locations.
-
-    Returns the path as a string or None if nothing was found, so the
-    return value can be used to test if the directory was found.
+    """Finds the directory for possible GRASS Database.
+    It automatically detects independent directory named grassdata 
+    in the usual locations. 
+    
+    Returns the path as a string or None. If nothing was found, 
+    the return value can be used to test if the directory was found.  
     """
-    home = os.path.expanduser('~')
+    home = os.path.expanduser('~')       
```

Comment:
Again, you have additional whitespace at the end of line, please make sure to get rid of that before you commit. git diff will show you whitespace changes. Extra whitespace is bad practice and it clutters the PR changes.

Reply:
@lindakladivova do you use a Python editor? If not, please consider to change - some of them are able to remove trailing white space automagically.

Reply:
I have switched on blank spaces in Spyder, so now hopefully, I will be able to see trailing white spaces.